queries.sql

```diff
@@ -31,3 +31,20 @@ WHERE name NOT IN ('Gabumon');
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts > 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
```

Comment:
**Changes Required**
- [ ] I don't seem to see the query for `Who escapes the most, neutered or not neutered animals?` Let us please add it